common/test/acceptance/tests/studio/test_studio_textbooks.py

```diff
@@ -29,6 +29,21 @@ def setUp(self, is_staff=True):
 
         self.textbook_view_page = TextbookViewPage(self.browser, self.course_id)
 
+    def _assert_textbook_data(self, textbook_data):
+        textbook_name = self.textbook_upload_page.textbook_name
```

Comment:
There is an actual diff-quality failure seems to be a missing docstring on this function.